sds_in_a_box/sds_in_a_box_stack.py

```diff
@@ -85,20 +97,48 @@ def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
                                           code=lambda_.Code.from_asset(os.path.join(os.path.dirname(os.path.realpath(__file__)), "SDSCode")),
                                           handler="indexer.lambda_handler",
                                           role=lambda_role,
-                                          runtime=lambda_.Runtime.PYTHON_3_9,
+                                          runtime=lambda_.Runtime.PYTHON_3_7,
                                           timeout=cdk.Duration.minutes(15),
                                           memory_size=1000,
                                           environment={
                                             "OS_ADMIN_USERNAME": "master-user", 
-                                            "OS_ADMIN_PASSWORD_LOCATION": os_secret.secret_name,
+                                            "OS_ADMIN_PASSWORD_LOCATION": os_secret.secret_value.unsafe_unwrap(),
```

Comment:
Is it ok to use the unsafe_unwrap() method for the two lambdas? Or does anyone have any better suggestions?

Reply:
The following issues were opened to address this comment:
https://github.com/IMAP-Science-Operations-Center/SDS-in-a-box/issues/22
https://github.com/IMAP-Science-Operations-Center/SDS-in-a-box/issues/23